utils/handleToasts.ts

```diff
@@ -3,24 +3,27 @@ import { toast } from 'react-toastify';
 export function handleErrorToast(errorText: string) {
   try {
     const toastExists = document.getElementById(errorText);
```

Comment:
Tällä rivillä pitää myös käyttää `toastId`:tä `errorText`in sijaan.

Reply:
Joo. Vähän liian innokkaasti etenin :see_no_evil: